config/task.yaml

```diff
@@ -44,6 +44,7 @@ spec:
       args:
       - publish
       - --image-refs=/tekton/results/IMAGES
+      - --arch=x86_64
```

Comment:
Why not all?

Reply:
I see: https://github.com/chainguard-dev/apko/pull/56#issuecomment-1061241005

Can we open an issue to tracking changing it to all?

Reply:
See above in the thread. Doing this for all archs requires some proot/qemu business that doesn't currently work. It's next.

Reply:
(as an aside, it'd be nice if there was support for binfmt_misc user namespaces in Linux, then we could just use unshare and not bother with proot)